src/main/resources/emissary/output/DropOffUtil.cfg

```diff
@@ -16,46 +16,49 @@ DATE_PARAMETER = "EventDate"
 DATE_PARAMETER = "FILE_DATE"
 
 # see SimpleDateFormat javadocs to understand these
-DATE_FORMAT = "E, d MMM yyyy HH:mm:ss Z"
-DATE_FORMAT = "E, d MMM yyyy HH:mm:ss z"
-DATE_FORMAT = "E, d MMM yyyy HH:mm:ss"
-DATE_FORMAT = "d MMM yyyy HH:mm:ss Z"
-DATE_FORMAT = "E, d MMM yy HH:mm:ss Z"
-DATE_FORMAT = "E, d MMM yyyy H:mm:ss Z"
-DATE_FORMAT = "E, d MMM yyyy HH:mm Z"
-DATE_FORMAT = "E, d MMM yyyy KK:mm:ss a Z"
-DATE_FORMAT = "E d MMM. yyyy HH:mm:ss Z"
-DATE_FORMAT = "E, d MMM. yyyy HH:mm:ss Z"
-DATE_FORMAT = "E, d MMM yyyy HH:mm:ss Z (z)"
-DATE_FORMAT = "E, MMM d, yyyy HH:mm:ss z"
-DATE_FORMAT = "E, MMM d, yyyy HH:mm:ss"
-DATE_FORMAT = "E, MMM d, yyyy KK:mm a"
-DATE_FORMAT = "dd-MMM-yyyy HH:mm:ss Z"
-DATE_FORMAT = "E dd-MMM-yyyy HH:mm:ss"
-DATE_FORMAT = "E dd-MMM-yyyy HH:mm:ss Z"
-DATE_FORMAT = "MM/dd/yy KK:mm a"
-DATE_FORMAT = "MM/dd/yy KK:mm:ss a"
-DATE_FORMAT = "MM/dd/yy KK:mm"
-DATE_FORMAT = "MM/dd/yy KK:mm:ss"
-DATE_FORMAT = "MM/dd/yy HH:mm"
-DATE_FORMAT = "MM/dd/yy HH:mm:ss"
-DATE_FORMAT = "MM/dd/yy kk:mm"
-DATE_FORMAT = "MM/dd/yy kk:mm:ss"
-DATE_FORMAT = "MMM d yyyy HH:mm:ss"
-DATE_FORMAT = "E MMM d HH:mm:ss yyyy"
-DATE_FORMAT = "E MMM d yyyy HH:mm:ss"
-DATE_FORMAT = "d MMM yyyy HH:mm z"
-DATE_FORMAT = "d MMM yyyy HH:mm"
-DATE_FORMAT = "E MMM d yyyy HH:mm:ss z"
-DATE_FORMAT = "E,d MMM yyyy KK:mm:ss a"
-DATE_FORMAT = "E, MMM d, yyyy, HH:mm:ss"
-DATE_FORMAT = "E, MMM d, yyyy, HH:mm:ss a"
-DATE_FORMAT = "E, MMM d, yyyy, HH:mm:ss a z"
-DATE_FORMAT = "E, MMM d, yyyy, HH:mm:ss a Z"
-DATE_FORMAT = "E, d MMM, yyyy HH:mm:ss a"
-DATE_FORMAT = "dd-MM-yyyy"
-DATE_FORMAT = "dd.MM.yyyy"
-DATE_FORMAT = "dd/MM/yyyy"
+DATE_TIME_OFFSET_FORMAT = "E[,][ ]d MMM[,][ ]uuuu[,][ ]HH:mm:ss Z"
```

Comment:
It looks like some of these patterns can be merged with the optional sections, but it's fine as is and this might be better for readability.